content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -93,9 +93,13 @@ set the `.spec.ipFamilyPolicy` field to one of the following values:
   * Selects the `.spec.ClusterIP` from the list of `.spec.ClusterIPs` based on the address family
     of the first element in the `.spec.ipFamilies` array.
 
+<<<<<<< HEAD
 If you would like to define which IP family to use for single stack or define the order of IP
 families for dual-stack, you can choose the address families by setting an optional field,
-`.spec.ipFamilies`, on the Service. 
+`.spec.ipFamilies`, on the Service.
+=======
+If you would like to define which IP family to use for single stack or define the order of IP families for dual-stack, you can choose the address families by setting an optional field, `.spec.ipFamilies`, on the Service.
+>>>>>>> cca4c78633 ([en] update en docs to use recommended labels)
```

Comment:
This looks like a manual merge gone wrong; please fix it.

Reply:
Apologies, I must have missed this during one of the rebases I had to do.